grammar.js

```diff
@@ -433,15 +433,15 @@ module.exports = grammar({
     file_redirect: $ => prec.left(seq(
       field('descriptor', optional($.file_descriptor)),
       choice('<', '>', '>>', '&>', '&>>', '<&', '>&', '>|'),
-      field('destination', repeat1($._literal)),
```

Comment:
I've added this because I don't think that a sequence of `1> foo bar > baz` is correct.